index.html

```diff
@@ -3,9 +3,11 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title><Etch-A-Sketch></title>
+        <script src="script.js"></script>
```

Comment:
Thanks for 

> Why is your script.js linked here and not within the body?

Because i feel since in my javascript i have added a  "DOMcontentloaded"  event listener which will allow my html and css to load first it makes no difference having the script in the head or body.
 Thanks sir